demos/defender/defender_demo_json/report_builder.c

```diff
@@ -86,8 +92,44 @@
     "\"total\": %u"              \
     "}"                          \
     "}"                          \
-    "}"                          \
     "}"
+
+#define JSON_REPORT_CUSTOM_METRIC_START \
```

Comment:
Can the following 4 macros be combined and thereby written in one `snprintf` call as opposed to 5 `snprintf` calls?

Reply:
Sure, done. I was trying to separate out the formats of _number-list_  and _string-list_ through macros to make the different formats easier for the reader to see.